### ISSUE TITLE
Does DiffDock recognize Chiral Centers?

### ISSUE THREAD

**Issue by rbjacob** [gcorso/DiffDock#55]
Hello, I have been trying to use DiffDock with Glufosinate specifically. It has a Chiral Center where one of the members is hydrogen. The smiles code for them is below.

R-glufosinate		CP(=O)(CC[C@H](C(=O)O)N)O 
L-glufosinate		CP(=O)(CC[C@@H](C(=O)O)N)O

When I converted these Smiles codes to SDF and ran DiffDock with the SDFs. The results were all of the same chirality. Is there some way to define the chirality differently. I am now running it with the smiles code itself rather than the SDF to see if there is a difference.

Thanks

**Comment by HannesStark** [comment]
The DiffDock model itself is able to distinguish different chiralities. 
We suspect there is a problem with the conversion of your smiles to structures. We recommend trying generating the sdf files yourself with the different chiralities and using them as input for diffdock.

**Comment by rbjacob** [comment]
Yes, I have played around with both input methods. What I found was that
when you submit a general smiles, there is no chirality. I found that when
I made and submitted Sdf files, there was no oreserved chirality. It was
only by submitting chiral specific smiles that the chirality of the
Molecule was preserved.

I suspect that there is a condition where the sdf files are made into
general smiles for further use in the software.
